uber/templates/preregistration/preregbase.html

```diff
@@ -18,7 +15,7 @@
     };
     setInterval(checkCap, 1000 * 1 * 90);
 
-{% if COLLECT_FULL_ADDRESS %}
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
You know, someday we're going to want to make a plugin for MAGFest Prime, because even different MAGFest events can't quite agree on core functionality.